### ISSUE TITLE
List automations/scenes/scripts that need updating when user changes entity_id

### ISSUE THREAD

**Issue by balloob** [home-assistant/home-assistant-polymer#4604]
## The request

When changing an entity ID in the entity settings dialog, let's list all scenes/scripts/automations that will need updating.

Something like "If you change this entity ID, you will need to update X, Y and Z!"

**Comment by dmulcahey** [comment]
Wouldn’t it be better to just go and update them for the user?

**Comment by balloob** [comment]
Yes, but let's start with this. We are limited in that we can only update the ones that are managed by the UI. We cannot change the ones defined in YAML.

**Comment by zsarnett** [comment]
Use search integration for this. MVP to display the automations/etc and the link to a page to edit?
